src/platform/qt/AboutDialog.ui

```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>AboutDialog</class>
+ <widget class="QWidget" name="AboutDialog">
+  <property name="windowModality">
+   <enum>Qt::WindowModal</enum>
```

Comment:
The window should not be modal. I couldn't even quit the program until dismissing it thanks to that!